src/templates/es/error.html

```diff
@@ -1,24 +1,24 @@
 {% extends "base/error.html" %}
 
-{% block unknown_error %}Error Desconocido{% endblock %}
+{% block unknown_error %}Error desconocido{% endblock %}
 
 {% 
   set localizedStatusCode = {
-    400: "Solicitud Incorrecta",
-    404: "Página no Encontrada",
-    500: "Error de Servidor Interno",
-    502: "Error Temporal del Servidor"
+    400: "Solicitud incorrecta",
+    404: "Página no encontrada",
+    500: "Error de servidor interno",
+    502: "Error temporal del servidor"
   }
 %}
 {% macro expandedTitle(errorTitle) %}Error: {{ errorTitle }}{% endmacro %}
 {%
   set localizedErrorMessage = {
-    "Unsupported year requested": "Año no Admitido Solicitado",
-    "Unsupported chapter requested": "Capítulo no Admitido Solicitado",
-    "Unsupported language requested": "Idioma no Admitido Solicitado",
+    "Unsupported year requested": "Solicitud de año no admitida",
+    "Unsupported chapter requested": "Solicitud de capítulo no admitida",
+    "Unsupported language requested": "Solicitud de idioma no admitida",
     "Not Found": "La URL solicitada no se encontró en el servidor. Si ingresó la URL manualmente, verifique su ortografía e intente nuevamente."
   }
 %}
 {% macro expandedErrorMessage(errorCode, errorDescription, errorMessage) %}
-{{ errorCode }} <i lang="en">{{ errorDescription }}</i>: {{ errorMessage }}
+{{ errorCode }} <i lang="es">{{ errorDescription }}</i>: {{ errorMessage }}
```

Comment:
This bit is actually in English (it's the original error code returned by the server).

```suggestion
{{ errorCode }} <i lang="en">{{ errorDescription }}</i>: {{ errorMessage }}
```